ocpmodels/trainers/energy_trainer.py

```diff
@@ -309,26 +308,13 @@ def train(self):
                                     disable_tqdm=False,
                                 )
 
-                        if self.is_hpo:
-                            progress = {
-                                "steps": current_step,
-                                "epochs": current_epoch,
-                                "act_lr": self.optimizer.param_groups[0]["lr"],
-                            }
-                            # checkpointing must be before reporter
-                            # report metrics to tune
-                            tune_reporter(
-                                iters=progress,
-                                train_metrics={
-                                    k: self.metrics[k]["metric"]
-                                    for k in self.metrics
-                                },
-                                val_metrics={
-                                    k: val_metrics[k]["metric"]
-                                    for k in val_metrics
-                                },
-                                test_metrics=None,
-                            )
+                        self.hpo_update(
```

Comment:
Could we put this `self.hpo_update` block inside an `if self.hpo:` block and remove that condition check from the `hpo_update` function in `BaseTrainer`?

Reply:
Yup, no problem.